.eslintrc

```diff
@@ -0,0 +1,23 @@
+{
+  "env": {
+    "browser": true,
+    "jest": true,
+    "es6": true
+  },
+  "plugins": [
+    "eslint-plugin-prettier"
+  ],
+  "extends": ["eslint:recommended", "plugin:react/recommended", "eslint-config-prettier"],
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
```

Comment:
nejake napady jaky config budeme pouzivat, napriklad airbnb?

Reply:
diky za odpoved,
tak to dava smysl i pro me. Poizival jsem z [tslint](https://github.com/cesko-digital/web/blob/rework/tslint.json) + `react/prop-types`.
Je to dobry napad pouzivat prop-types?

Reply:
Za mne ano. 👍  ... V budoucnu samozřejmě prop-types nahradíme TS, ale prozatím klidně můžeme přidat.

Reply:
dobre, tak odstranit ten rule a pridam prop-types pro componenty